src/main/kotlin/no/nav/tilleggsstonader/sak/vilkår/vilkårperiode/AldersvilkårVurdering.kt

```diff
@@ -0,0 +1,93 @@
+package no.nav.tilleggsstonader.sak.vilkår.vilkårperiode
+
+import no.nav.tilleggsstonader.sak.infrastruktur.exception.feilHvis
+import no.nav.tilleggsstonader.sak.opplysninger.grunnlag.Grunnlagsdata
+import no.nav.tilleggsstonader.sak.vilkår.vilkårperiode.domain.MålgruppeType
+import no.nav.tilleggsstonader.sak.vilkår.vilkårperiode.domain.faktavurderinger.SvarJaNei
+import no.nav.tilleggsstonader.sak.vilkår.vilkårperiode.dto.LagreVilkårperiode
+import java.time.LocalDate
+
+object AldersvilkårVurdering {
+    fun vurderAldersvilkår(
+        vilkårperiode: LagreVilkårperiode,
+        grunnlagsData: Grunnlagsdata,
+    ): SvarJaNei {
+        val fødselsdato = grunnlagsData.grunnlag.fødsel?.fødselsdato
```

Comment:
Alle har ikke fødselsdato, då kan vi eventuellt bruke denne selv om det kanskje ikke er helt korrekt å bruke 1 jan
```suggestion
        val fødselsdato = grunnlagsData.grunnlag.fødsel?.fødselsdatoEller1JanForFødselsår()
```

Reply:
Kan ev spørre en jurist om dette

Reply:
Har hørt med Marius i AAP, og det kaster feilmelding dersom bruker mangler fødselsdato. Tenker det er fornuftig at vi legger på samme linje.